examples/sveltekit/src/routes/(app)/+page.svelte

```diff
@@ -27,7 +27,7 @@
 		{/if}
 
 		{#if $page.url.searchParams.get('auth-type') === 'magiclink'}
-			<form action="/?/create-magiclink" method="post" use:enhance={handleSubmit}>
```

Comment:
🥲 x2

Reply:
Another unrelated change, this should have been in a different PR but it was a necessary change during testing.